gravitee-node-api/src/main/java/io/gravitee/node/api/upgrader/Initializer.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.node.api.upgrader;
+
+/**
+ * @author Kamiel Ahmadpour (kamiel.ahmadpour at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+// Initializer will run each upgrade
+public interface Initializer {
```

Comment:
I though the Initializer was put outside of the upgrader scope 🧐
Can it be at least in another package than `io.gravitee.node.api.upgrader` to make it really distinct (initializer ?)

Reply:
Eventually we decided to keep it with David :)

Reply:
Sounds good to me as long as we don't have an `Initializer` interface in a `upgrader` package ;-)